src/Domain/Entities/Characters/CharacterStatistics.cs

```diff
@@ -1,9 +1,12 @@
-﻿namespace Crpg.Domain.Entities.Characters;
+﻿using Crpg.Domain.Entities.Servers;
+
+namespace Crpg.Domain.Entities.Characters;
 
 public class CharacterStatistics
 {
     public int Kills { get; set; }
     public int Deaths { get; set; }
     public int Assists { get; set; }
     public TimeSpan PlayTime { get; set; }
+    public GameMode GameMode { get; set; }
 }

```

Comment:
You need to add a navigation property:
`public Character? Character{ get; set; }`


After that, you need to delete the old migration and create a new migration

ref1:
```
src/Domain/Entities/Parties/PartyItem.cs
```
ref2: https://learn.microsoft.com/en-us/ef/ef6/fundamentals/relationships